lumicks/pylake/nb_widgets/kymotracker_widgets.py

```diff
@@ -732,57 +733,98 @@ def _get_default_parameters(kymo, channel):
+            "int",
+            4,
+            *(1, 15),
+            True,
+            r"Maximum gap duration defines how big the gap between two points can be for them to "
+            r"connect. It should be chosen such that small gaps in a track can be overcome and "
+            r"tracked as one, but not so large that separate tracks are strung together.",
         ),
-        "sigma": KymotrackerParameter(
-            "Sigma",
-            "How much does the track fluctuate?",
+        "min_length": KymotrackerParameter(
+            "Minimum length",
+            "Minimum number of frames a spot has to be detected in to be considered.",
+            "int",
+            3,
+            *(1, 10),
+            True,
+            "Minimum length defines the minimum number of points a tracked line must have for it "
+            "to show up. Reducing this parameter is very effective in reducing noise. Note that "
+            "this length refers to the number of detections, not length in time!",
+        ),
+        "track_width": KymotrackerParameter(
+            "Expected spot size",
+            "How big a particle needs to appear to be tracked as a single molecule.",
             "float",
-            2 * position_scale,
-            *(1.0 * position_scale, 5.0 * position_scale),
+            4 * position_scale,
+            *(0.0, 15.0 * position_scale),
             True,
+            r"Minimum particle width defines how big a particle needs to appear as on the "
+            r"kymograph for it to be tracked as a single molecule. This parameter should be set to "
+            r"roughly the width of the point spread function. Making it larger rejects more noise, "
+            r"but at the cost of potentially merging tracks that are close together.",
         ),
         "vel": KymotrackerParameter(
-            "Velocity",
+            "Expected velocity",
             "How fast does the particle move?",
             "float",
             0.0,
             *(-5.0 * vel_calibration, 5.0 * vel_calibration),
             True,
+            "Minimum particle velocity defines how fast particles can move on along the DNA on "
+            "average. When tracking, the algorithm searches for points in future scan lines to "
+            "connect. Points within a certain distance from the expected future position are "
+            "connected.",
         ),
         "diffusion": KymotrackerParameter(
-            "Diffusion", "Expected diffusion constant.", "float", 0.0, *(None, None), False
+            "Diffusion",
+            "Expected diffusion constant.",
+            "float",
+            0.0,
+            *(None, None),
+            False,
+            "When tracking, the algorithm searches for peaks in future frames to connect. Peaks "
+            "within a certain distance from the expected future position are connected. The "
+            "diffusion parameter determines how quickly this distance grows over time.",
```

Comment:
Not super important since this one isn't displayed, but just for consistency:

```suggestion
            "When tracking, the algorithm searches for points in future frames to connect. Points "
            "within a certain distance from the expected future position are connected. The "
            "diffusion parameter determines how quickly this distance grows over time.",
```

Reply:
👍